Controller/Adminhtml/SaveWidget/Index.php

```diff
@@ -86,9 +86,9 @@ public function execute()
         $response = $this->resultJsonFactory->create();
         $response->setHeader('Content-type', 'application/json');
 
-        $pageId = $this->request->getParam('pageId');
-        $widgetId = $this->request->getParam('widgetId');
-        $storeId = $this->request->getParam('id');
+        $pageId = strip_tags($this->request->getParam('pageId'));
+        $widgetId = strip_tags($this->request->getParam('widgetId'));
+        $storeId = preg_replace("/['\"]/", "", strip_tags($this->request->getParam('id')));
```

Comment:
`preg_replace("/['\"]/", "", strip_tags(` can move into helper function, so that it has consistent implementation

Reply:
added helper function